zowe/zowe-cli
# Update tsconfig and web help deps

⚠️ Don't merge until https://github.com/zowe-actions/octorelease/pull/147 is published

**What It Does**

* Enable declaration maps in tsconfig so that "Go to Definition" goes to `.ts` files instead of `.d.ts` files
* Update web help dependencies for technical currency (including major update from Bootstrap 4 to 5)
* Fix active node not updating in web help tree when browsing as a local file

**How to Test**

To test the tsconfig fix:
1. Check out this branch and don't rebuild yet.
1. Open a source file that imports classes from other SDKs (such as `packages/zosuss/src/SshBaseHandler.ts`).
1. Ctrl+Click on one of the imported class names (e.g. `ImperativeError`) and notice a `.d.ts` file is opened.
1. Rebuild , then Ctrl+Click the class name again and notice a `.ts` file is opened.

To test changes to web help, click on links in the doc and verify that active (highlighted) item in tree updates:
![test-web-help-cropped](https://github.com/user-attachments/assets/8472cfe9-e653-4285-aee9-be17cefdf72a)


**Review Checklist**
I certify that I have:


**Additional Comments**
